pkg/types/azure/defaults/machines.go

```diff
@@ -0,0 +1,20 @@
+package defaults
+
+import (
+	"fmt"
+)
+
+func BootstrapMachineType(region string) string {
+	instanceClass := InstanceClass(region)
+	return fmt.Sprintf("%s_D4s_v3", instanceClass)
+}
+
+func MasterMachineType(region string) string {
```

Comment:
can you rename to `ControlPlaneInstanceType`

Reply:
Renamed!